exasol/bucketfs/__init__.py

```diff
@@ -115,25 +115,38 @@ class Service:
         buckets: lists all available buckets.
     """
 
-    def __init__(self, url: str, credentials: Mapping[str, Mapping[str, str]] = None):
+    def __init__(
+        self,
+        url: str,
+        credentials: Mapping[str, Mapping[str, str]] = None,
+        verify: bool | str = True,
+    ):
         """Create a new Service instance.
 
         Args:
-            url: of the bucketfs service, e.g. `http(s)://127.0.0.1:2580`.
-            credentials: a mapping containing credentials (username and password) for buckets.
+            url:
+                Url of the bucketfs service, e.g. `http(s)://127.0.0.1:2580`.
+            credentials:
+                A mapping containing credentials (username and password) for buckets.
                 E.g. {"bucket1": { "username": "foo", "password": "bar" }}
+            verify:
+                Either a boolean, in which case it controls whether we verify
+                the server's TLS certificate, or a string, in which case it must be a path
+                to a CA bundle to use. Defaults to ``True``.
```

Comment:
```suggestion
                Either a boolean, in which case it controls whether we verify
                the server's TLS certificate (False, verifiation disabled, True, verification uses default CAs), or a string, in which case it must be a path
                to a CA bundle to use. Defaults to ``True``.
```

Reply:
I would like to leave it as is for consistency with the `requests` docs.